frontend/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "pec-frontend",
+  "version": "0.1.0",
```

Comment:
As far as I know there are plenty of fields in package.json schema which are available to provide rich metadata for project and they are not present here. 
![image](https://github.com/UniversityOfGdanskTeamPython/pec_framework_web_app/assets/56170852/e16ec167-4573-412c-b40f-a38e975a604b)
[package.json](https://docs.npmjs.com/cli/v10/configuring-npm/package-json) docs

Reply:
metadata is now rich in information fields